osmaxx-py/osmaxx/excerptexport/templates/excerptexport/templates/login.html

```diff
@@ -19,9 +19,27 @@
     <div>{{ form.password }}</div>
 </div>
 
-<input type="submit" value="login" />
-<input type="hidden" name="next" value="{{ next }}" />
+    <input type="submit" value="login" />
+    <input type="hidden" name="next" value="{{ next }}" />
```

Comment:
If you indent these lines because they are nested in a `<form>` element, the other lines in the same element (13-20) should be indented, too.